Lib/csv.py

```diff
@@ -71,6 +71,7 @@ class excel:
                  QUOTE_STRINGS, QUOTE_NOTNULL
 from _csv import Dialect as _Dialect
 
+from collections import defaultdict, Counter
```

Comment:
Can you use a local import within `Sniffer`? `collections` is quite a heavy module to import.